src/main/java/ladder/model/Players.java

```diff
@@ -1,43 +1,59 @@
 package ladder.model;
 
+import ladder.exceptionMessage.ExceptionMessage;
+
 import java.util.Collections;
 import java.util.List;
+import java.util.Optional;
 
 public class Players {
 
     private static final int MIN_PLAYER_COUNT = 2;
+    private static final int MAX_PLAYER_COUNT = 26;
 
     private final List<Player> players;
 
     public Players(List<Player> players) {
         validatePlayerCount(players);
+        validateDuplicatedNames(players);
         this.players = players;
     }
 
     private void validatePlayerCount(List<Player> players) {
-        if (players.size() < MIN_PLAYER_COUNT) {
-            throw new IllegalArgumentException(ErrorMessage.EXCEPTION_INVALID_PLAYER_COUNT.getMessage());
+        if (!isPlayerCountIncludedInRange(players.size())) {
+            throw new IllegalArgumentException(ExceptionMessage.EXCEPTION_INVALID_PLAYER_COUNT.getMessage());
         }
     }
 
-    public int size() {
-        return players.size();
+    private void validateDuplicatedNames(List<Player> names) {
+        if (names.size() != names.stream().distinct().count()) {
+            throw new IllegalArgumentException(ExceptionMessage.EXCEPTION_DUPLICATED_NAME.getMessage());
+        }
     }
 
-    public List<Player> getPlayers() {
-        return Collections.unmodifiableList(players);
+    private boolean isPlayerCountIncludedInRange(int playerCount) {
+        return MIN_PLAYER_COUNT <= playerCount && playerCount <= MAX_PLAYER_COUNT;
     }
 
-    private enum ErrorMessage {
-        EXCEPTION_INVALID_PLAYER_COUNT("게임을 진행하기 위해서는 두 명 이상의 플레이어가 필요합니다.");
-        private final String message;
+    public Player findPlayer(Player givenPlayer) {
+        Optional<Player> foundPlayer = Optional.ofNullable(players.stream()
+                .filter(player1 -> player1.equals(givenPlayer))
+                .findFirst()
+                .orElseThrow(() -> new IllegalArgumentException(ExceptionMessage.EXCEPTION_INVALID_PLAYER_NAME.getMessage())));
 
-        ErrorMessage(String message) {
-            this.message = message;
-        }
+        return foundPlayer.get();
```

Comment:
```suggestion
public Player findPlayer(Player givenPlayer) {
        return players.stream()
                .filter(player1 -> player1.equals(givenPlayer))
                .findFirst()
                .orElseThrow(() -> new IllegalArgumentException(ExceptionMessage.EXCEPTION_INVALID_PLAYER_NAME.getMessage()));
```

findFirst() 후에 나오는 Optional.empty()는 orElseThrow()가 예외 처리하니까 이렇게 리팩토링할 수 있지 않을까요?